simple_history/models.py

```diff
@@ -545,7 +545,10 @@ def transform_field(field):
 
     elif isinstance(field, models.FileField):
         # Don't copy file, just path.
-        field.__class__ = models.TextField
+        if getattr(settings, "SIMPLE_HISTORY_FILEFIELD_TO_CHARFIELD", False):
+            field.__class__ = models.CharField
```

Comment:
Does this not fail if you don't specify max length?

Reply:
It works because FileFields [default to](https://github.com/django/django/blob/master/django/db/models/fields/files.py#L239) max_length of 100. 

django-simple-history's test model `FileModel` doesn't have an explicit `max_length`, for example. Here's a pdb session from inside `HistoricalRecordsTest.test_file_field` with the setting forced to True:

```
>>> FileModel.history.model._meta.get_field('file')
<django.db.models.fields.CharField: file>
>>> FileModel.history.model._meta.get_field('file').max_length
100
```

Reply:
Awesome. Mind just adding a test for when the setting is true?

Reply:
Done! Would have included tests before, but it took me a while to find the clever way of mocking model-setup settings in tests/models.py.